english/chess/CheesePuzzle.md

```diff
@@ -0,0 +1 @@
+[it sort of makes me want to cry...](http://2.bp.blogspot.com/-8VIW5EHOXis/VC6PctbH9kI/AAAAAAAAM0s/n_5JtDdrhQY/s1600/IMG_3434.JPG)
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.